libs/clients/rsk-company-info/src/openapi.yml

```diff
@@ -20,6 +20,21 @@ paths:
           schema:
             type: string
             default: Miracle%20ehf
+        # TODO: Limit and offset were manually added in. Update accordingly after RSK has updated their openapi to include limit and offset
+        - name: limit
+          in: query
+          required: false
+          description: Limit the number of results
+          schema:
+            type: string
+            default: '100'
```

Comment:
You can see some examples of scripted modifications in workspace.json "update-openapi-document" commands using `jq` black magic. But ideally they fix this on their side.

Don't need to stress over this comment. Just an FYI.